cloudhub-2/modules/ROOT/pages/ch2-manage-schedules.adoc

```diff
@@ -25,27 +25,36 @@ To view the application schedules, you must have the *Exchange Viewer* permissio
 
 When designing the schedule, consider the following:
 
+
+* Plan for high availability during network failures. CloudHub 2.0 infrastructure restarts due to hardware issues or scheduled maintenance, and security updates.
++
+During service updates or security patching, xref:mule-runtime::global-settings-configuration.adoc#global-configurations-reference[CloudHub 2.0 waits five minutes] for existing schedules to complete. After a new replica is launched, the schedule runs at its next due. 
+* Design your schedules to prevent duplicate data processing, as they can be triggered more than once due to the behavior mentioned previously.
```

Comment:
Specifically which previously mentioned behavior? 

Should this be something like:
* Because schedules can be triggered more than once, design them to prevent duplicate data processing.

For example, what would they do to prevent this?